client/customer-effort-score-tracks/customer-effort-score-helper.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * External dependencies
+ */
+import { render } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import CustomerEffortScoreTracks from './customer-effort-score-tracks';
+
+let cesSurveyForAnalyticsShown = false;
+export function showCesSurveyForAnalytics() {
+	if ( cesSurveyForAnalyticsShown ) {
+		return;
+	}
+	cesSurveyForAnalyticsShown = true;
+
+	const appRoot = document.getElementById( 'root' );
+	const embeddedRoot = document.getElementById( 'woocommerce-embedded-root' );
+	const root = appRoot || embeddedRoot;
+
+	render(
+		<CustomerEffortScoreTracks
```

Comment:
Pardon my ignorance on this feature as this is my first time diving into this code.

Is there a reason we can't use the existing `core/notices` that are displayed in the `TransientNotices` component?  This takes advantage of the existing stores and doesn't require the addition of new render or mount points.

If we're worried about namespacing the notices, it could be a good time to introduce `woocommerce/notices` as a store.

Reply:
@joshuatf I think it's because we have initialization logic that `core/notices` does not support as shown as [here](https://github.com/woocommerce/woocommerce-admin/blob/update/5349-trigger-ces-on-filtering-analytics/client/customer-effort-score-tracks/customer-effort-score-tracks.js), but I think @becdetat is the best person to answer for the `CustomerEffortScoreTracks` component.